crates/re_log/src/setup.rs

```diff
@@ -3,57 +3,61 @@
 /// Directs [`log`] calls to stderr.
 #[cfg(not(target_arch = "wasm32"))]
 pub fn setup_native_logging() {
```

Comment:
why do we even distinguish native logging and web logging methods. Could just be one 🤔

Reply:
that... is a good point :sweat_smile: lemme see

Reply:
Yup, looks good; good call :+1: